package.json

```diff
@@ -12,6 +12,7 @@
     "kind": "plugin"
   },
   "dependencies": {
+    "@strapi/helper-plugin": "4.1.9",
```

Comment:
Shouldn't we be getting this package from Strapi itself?

Reply:
@melishev right, I have removed the package and used it from Strapi.